src/current/v24.3/architecture/replication-layer.md

```diff
@@ -146,6 +146,33 @@ A table's meta and system ranges (detailed in the [distribution layer]({% link {
 
 However, unlike table data, system ranges cannot use epoch-based leases because that would create a circular dependency: system ranges are already being used to implement epoch-based leases for table data. Therefore, system ranges use expiration-based leases instead. Expiration-based leases expire at a particular timestamp (typically after a few seconds). However, as long as a node continues proposing Raft commands, it continues to extend the expiration of its leases. If it doesn't, the next node containing a replica of the range that tries to read from or write to the range will become the leaseholder.
 
+#### Leader leases
+
+{% include_cached new-in.html version="v24.3" %}
+
+{% include feature-phases/preview.md %}
+
+{% include {{ page.version.version }}/leader-leases-intro.md %}
+
+Leader leases rely on a shared, store-wide failure detection mechanism for triggering new Raft elections. [Stores]({% link {{ page.version.version }}/cockroach-start.md %}#store) participate in Raft leader elections by "fortifying" a candidate replica based on that replica's store liveness, as determined among a quorum of all the node's stores. A replica can **only** become the Raft leader if it is so fortified.
```

Comment:
Is store liveness a new concept that is distinct from node liveness? If so, it might be worth clarifying that point. I assumed this was node liveness at first. The simplest way to do that would just be to italicize the term.

Reply:
✅